.github/workflows/docs.yml

```diff
@@ -150,6 +169,25 @@ jobs:
           publish_dir: doc/_build/html/
           cname: docs.pyvista.org
 
+      - name: Remove non-multi version folder
+        run: rm -rf doc
+
+      - name: Deploy on main multiversion
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: pyansys/actions/doc-deploy-dev@main
```

Comment:
Is it appropriate to use PyAnsys action here? I believe 100% in the PyAnsys community, but this action is an action for PyAnsys and maybe change in the future. How about forking the action from PyAnsys action and maintaining the action independently in the PyVista community? (If OK, I will fork and maintain.).

Reply:
Thanks for raising this question, @tkoyama010. This is a decission you need to agree in PyVista.

We refactored PyAnsys actions according to the feedback of @banesullivan. The only requirements imposed by the documentation deployment actions are:

* Use a theme that inherits from the `pydata-sphinx-theme`
* Apply tags following a `v<Major>.<Minor>.<Patch>` semantic version

I would suggest to go with pyansys/actions for the moment, as they are fully compatible.

If at some point these actions become a limitation, open an issue. If the issue gets discarded, fork and adapt to PyVista needs.

Reply:
Thank you. We will address the issue when it arises.